### ISSUE TITLE
Allow signed-in users to submit feedback anonymously

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-frontend#12532]
Signed-in should be able to report feedback anonymously. In order to do that we'll need to change the form but also avoid forwarding the credentials when we POST the data to the API.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/ADDFRNT-258)

**Comment by willdurand** [comment]
<img width="795" alt="Screenshot 2023-11-07 at 12 08 20" src="https://github.com/mozilla/addons-frontend/assets/217628/3d0919ab-7ead-4a33-94d4-4fbc541884b8">

**Comment by ioanarusiczki** [comment]
@willdurand 

The bad news is that I hit the 24 hour IP throttle , also  good news since it's expected. I'll check this tomorrow

 
![IP throttle](https://github.com/mozilla/addons-frontend/assets/33448286/a6c6b3aa-283e-43ad-840f-a21306362bfe)

**Comment by willdurand** [comment]
cc @diox

**Comment by diox** [comment]
I reset the IP throttle for @ioanarusiczki

**Comment by ioanarusiczki** [comment]
@willdurand 

1. When the File report anonymously is checked, reporter field in admin is empty which means that the user information was not sent.

![anonim](https://github.com/mozilla/addons-frontend/assets/33448286/d4b595e6-dbb3-4f85-800f-cbdce6c2f27d)


2. I've noticed for unauthenticated users that the form can be sent with or without checking this option. When checked it would grey out the Name and e-mail fields. Is this intended or it should require Name and e-mail when not checking the box?

![no login in the browser](https://github.com/mozilla/addons-frontend/assets/33448286/76d12356-4130-48df-8708-0659ef482c01)

**Comment by willdurand** [comment]
> I've noticed for unauthenticated users that the form can be sent with or without checking this option. When checked it would grey out the Name and e-mail fields. Is this intended or it should require Name and e-mail when not checking the box?

Yeah, I guess for unauthenticated users, we might not need the `File report anonymously`.

**Comment by willdurand** [comment]
> Yeah, I guess for unauthenticated users, we might not need the `File report anonymously` checkbox.

Scratch that, talking with UX, we should make the input fields required instead. I'll file a new issue for that.

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDFRNT-258
